Assets/Dev/B/Script/getObjectonTile.cs

```diff
@@ -1,19 +1,35 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
 
```

Comment:
This script is still named incorrectly. Try to delete it entirely in one commit, and make a new one with the right name in the next commit.

Reply:
Done